### PR TITLE
#1:Added initial setup files

### LINKED ISSUE #1: Inital Setup of the Project
- add gtihub workflows
- add githooks
- a short README.rst
- doc folder including changes folder, guides... etc
- poetry setup
- subdirectories such as lu/src, lua/test

### PR DESCRIPTION
**Pull request by umitbuyuksahin**
fixes #1

### CODE REVIEW

File: .github/workflows/release_droid_prepare_original_checksum.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Release Droid - Prepare Original Checksum
+
+on:
+  workflow_dispatch:
```
Comment (tkilias) [inline .github/workflows/release_droid_prepare_original_checksum.yml:4]: I think, this file is not necessary anymore for release droid

File: .github/workflows/release_droid_print_quick_checksum.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Release Droid - Print Quick Checksum
+
+on:
```
Comment (tkilias) [inline .github/workflows/release_droid_print_quick_checksum.yml:3]: I think, this file is not necessary anymore for release droid

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "exasol-advanced-analytics-framework"
+version = "0.1.0"
+description = "Framework for data analysis methodologies"
```
Comment (tkilias) [inline pyproject.toml:4]:
```suggestion
description = "Framework for building complex data analysis algorithms with Exasol"
```

**Review by tkilias — DISMISSED** [review]

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
+
+# Poetry
+poetry.lock
```
Comment (MarleneKress79789) [inline .gitignore:132]: I think dependabot finds dependencies issues in poetry.lock. Should we commit it, like int some other python repos?

File: .github/workflows/pytest.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+name: Run pytest tests
+
+on: [ push, pull_request ]
+
+jobs:
+  integration_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```
[the remark this answers is not in the document]
Comment (umitbuyuksahin) [reply .github/workflows/pytest.yaml:10]: it would be better to make it 3.8. thanks

**Review by MarleneKress79789 — APPROVED** [review]

**Pull request merged**